aws/src/integration/java/org/apache/iceberg/aws/lakeformation/TestLakeFormationMetadataOperations.java

```diff
@@ -241,7 +241,7 @@ public void testAlterTableSetPropertiesNoDataPathAccess() {
       AssertHelpers.assertThrows("attempt to alter a table without ALTER permission should fail",
           ForbiddenException.class,
           "Glue cannot access the requested resources",
-          () -> updateProperties.commit());
```

Comment:
Question (non-blocking): What warning does this give (if you remember)?

Reply:
```
/...iceberg/aws/src/integration/java/org/apache/iceberg/aws/lakeformation/TestLakeFormationMetadataOperations.java:244: warning: [LambdaMethodReference] Lambda should be a method reference
          () -> updateProperties.commit());
          ^
    (see https://github.com/palantir/gradle-baseline#baseline-error-prone-checks)
```